config/filter.d/drupal-auth.conf

```diff
@@ -0,0 +1,28 @@
+# Fail2Ban filter to block repeated failed login attempts to Drupal site(s)
+#
+#
+# Drupal must be setup to use Syslog, which defaults to the following format:
+#
+#   !base_url|!timestamp|!type|!ip|!request_uri|!referer|!uid|!link|!message
+#
+#
+
+[INCLUDES]
+
+before = common.conf
+
+
+[Definition]
+
+_daemon =
+
+failregex = ^%(__prefix_line)s.+\|\d{10}\|user\|<HOST>\|.+\|.+\|\d\|.?\|Login attempt failed for .+\.$
```

Comment:
`<HOST>` is floating inbetween `.+`s -- not good... I smell possible injection especially since we are monitoring syslog . Please make leading `.+` way more specific (pattern for url with no spaces etc)